Dima/netflix-roulette/src/interfaces/IState.ts

```diff
@@ -0,0 +1,4 @@
+import IFilm from './IFilm';
+export default interface IState {
+  films: IFilm[],
```

Comment:
No need in separate interface for array. Usually we just use IFilm[] where it's needed